kronor_swish/src/main/java/io/kronor/component/swish/SwishComponent.kt

```diff
@@ -317,31 +322,34 @@ fun swishAppExists(context: Context): Boolean {
 
 @Composable
 fun SwishPromptMethods(onAppOpen: () -> Unit, onQrCode: () -> Unit, onPhoneNumber: () -> Unit) {
-    val doesSwishAppExist: Boolean = if (LocalInspectionMode.current) {
-        true
-    } else {
-        swishAppExists(LocalContext.current)
-    }
+    val doesSwishAppExist: Boolean =
+        LocalInspectionMode.current.takeIf { true } ?: swishAppExists(LocalContext.current)
```

Comment:
`this.takeIf p = if (p(this)) then this else null`

but what we want is

`this.takeIf p = if (this) then p else null`

So please revert this change

Reply:
not sure why that happened. I have changed it back to the old version for now.